packages/focusvisible/stories.tsx

```diff
@@ -14,7 +14,11 @@ import { withKnobs } from '@storybook/addon-knobs';
 import { useFocusVisible, FocusVisibleContainer } from './src';
 import { useSelection } from '../selection/src';
 
-const StyledCustomFocus = styled.div`
+interface IStyledCustomFocus extends React.HTMLProps<HTMLDivElement> {
+  isSelected?: boolean;
```

Comment:
Even though this matches the previous story, I'm not sure we are using the `isSelected` value for anything valid? Maybe include some dummy styling similar to the `useSelection()` story.

Reply:
`isSelected` is used for styling the border bottom of the selected element in the "Usage with useSelection" example [here](https://github.com/zendeskgarden/react-containers/blob/master/packages/focusvisible/stories.js#L91). I think it's needed since the styling is based on the selected state for that example, but would love to get your thoughts.